### PR TITLE
feat: Add Azure instance table on Launch success

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Adds Instance table at the Launch success screen.

Adds instance link humanize function, as for Azure, we want shorten the visible ID.

Allows passing default values overrides for the WizardContext, this enables testing with different providers easily.

Refs HMS-1409

Screenshot from 2023-04-11 17-35-59

### CODE REVIEW

File: src/Components/Common/instanceHelpers.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { AWS_PROVIDER, AZURE_PROVIDER } from './constants';
+
+export const instanceLink = (instanceID, provider, region) => {
+  switch (provider) {
+    case AWS_PROVIDER:
+      return `https://console.aws.amazon.com/ec2/home?region=${region}#InstanceDetails:instanceId=${instanceID}`;
+    case AZURE_PROVIDER:
+      return `https://portal.azure.com/#@rhdevcloudops.onmicrosoft.com/resource${instanceID}/overview`;
```
Comment (lzap) [inline src/Components/Common/instanceHelpers.js:8]: It reads `resourceXXXX` immediately without any separator, is that correct?

Comment (lzap) [reply src/Components/Common/instanceHelpers.js:8]: Also the tenant is hardcoded?

File: fec.config.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 const { resolve } = require('path');
 
 module.exports = {
-  appUrl: '/apps/provisioning',
+  appUrl: '/insights/provisioning',
```
Comment (lzap) [inline fec.config.js:5]: Is this intentional?

File: src/Components/Common/instanceHelpers.js (hunk 1 above)
Comment (ezr-ondrej) [reply src/Components/Common/instanceHelpers.js:8]: Yeah, it is the Azure instanceID is of a format (I'm simplifying - but it starts with `/`) `/subscription/<sub_id>/resource_group/<resource_group_name>/Microsoft.Compute/<instance_name>`

**Review by lzap — APPROVED** [review]
Other than that...

File: src/Components/Common/instanceHelpers.js (hunk 1 above)
Comment (lzap) [reply src/Components/Common/instanceHelpers.js:8]: Gotcha and our tenant is hardcoded because that is the service account I guess.

**Pull request merged**